tests/bluetooth/audio/ascs/prj.conf

```diff
@@ -5,11 +5,10 @@ CONFIG_BT_MAX_CONN=1
 CONFIG_BT_ISO_MAX_CHAN=1
 CONFIG_BT_AUDIO=y
 CONFIG_BT_ASCS=y
-CONFIG_BT_ASCS_ASE_SNK_COUNT=2
-CONFIG_BT_ASCS_ASE_SRC_COUNT=2
+CONFIG_BT_ASCS_ASE_SNK_COUNT=1
+CONFIG_BT_ASCS_ASE_SRC_COUNT=1
```

Comment:
We don't have any tests that rely on multiple streams?

Reply:
Nope

Reply:
We should consider that at some point - I guess none of the for loops to handle multiple ASEs in a single GATT write command is properly exercised currently